lesson15/js/main.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+const openModalBtn = document.getElementById('js-openModalBtn');
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const getModal = document.getElementById('js-modalContainer');
+		getModal.classList.add('is-visible');
+		document.body.insertAdjacentHTML(
+			'afterbegin',
+			'<div id="js-bgFixed" class="el_modalBg"></div>'
+		);
+		document.body.style.position = 'fixed';
+	},
+	false
+);
+
+function closeModal() {
+	const getModal = document.getElementById('js-modalContainer');
+	getModal.classList.remove('is-visible');
+	document.body.removeAttribute('style');
+	const modalBg = document.getElementById('js-bgFixed');
+	modalBg.remove();
+}
+
+const closeModlBtn = document.getElementById('js-closeModal');
+closeModlBtn.addEventListener(
+	'click',
+	() => {
+		closeModal();
+	},
+	false
+);
+
+function hideModalButton() {
+	const openModalBtn = document.getElementById('js-openModalBtn');
+	openModalBtn.remove();
+}
+
+const form = document.getElementById('js-loadmenuBtn');
+form.addEventListener(
+	'click',
+	(event) => {
+		event.preventDefault();
+		const inputNumber = document.getElementById('number');
+		const inputText = document.getElementById('words');
+		if (inputNumber.value && inputText.value && inputText.value.trim()) {
```

Comment:
IMO
Since trim() returns a string with the whitespace trimmed from both ends and we want to know the boolean value of that value, I think it is more appropriate to write it as follows

```suggestion
		if (inputNumber.value && inputText.value.trim()) {
```

Reply:
Thank you!!!!
Certainly Sae is right!!!

I thought that it was correct to judge inputText after looking into the extra space with the trim () function, so I modified it.

[36610c8](https://github.com/nyattsu72/hanson-frontend-cource/pull/20/commits/36610c81c0d6eed0a554f0e726d37f235ef25ab2)